src/electionguard/facades/manifest.cpp

```diff
@@ -1994,7 +1996,8 @@ eg_electionguard_status_t eg_election_manifest_from_json(char *in_data,
         *out_handle = AS_TYPE(eg_election_manifest_t, deserialized.release());
         return ELECTIONGUARD_STATUS_SUCCESS;
     } catch (const exception &e) {
-        Log::error(":eg_election_manifest_from_json", e);
+        Log::error(__func__, e);
+        ExceptionHandler::getInstance().setData(__func__, ELECTIONGUARD_STATUS_ERROR_BAD_ALLOC, e);
```

Comment:
what do you think about just coupling this with the Log class and adding the setData call to all calls to Log::error?

Reply:
Added